src/lib/features/events/event-store.ts

```diff
@@ -89,8 +89,8 @@ export interface IEventTable {
 
 export interface IEventSearchParams {
     query: string | undefined;
-    offset: string | undefined;
-    limit: string | undefined;
+    offset: number | undefined;
+    limit: number | undefined;
```

Comment:
Why do we have two interfaces with the exact same name now? Did we always? Is it gonna cause confusion?

Reply:
Good catch, removed one.